src/containers/Ecotip/Ecotip.css

```diff
@@ -0,0 +1,14 @@
+.ecotip-card {
```

Comment:
Great choice in rule value measurements!  This will be great for the responsiveness of the application

Reply:
Thank you!